src/openms_gui/source/VISUAL/TVIdentificationViewController.cpp

```diff
@@ -309,14 +312,17 @@ namespace OpenMS
     Plot1DWidget* widget_1D = tv_->getActive1DWidget();
 
     // return if no active 1D widget is present
-    if (widget_1D == nullptr)
-    { 
-      return;
+    if (widget_1D == nullptr) 
+    {
+      std::cout << "Current widget is nullptr" << std::endl;
+      return; 
     }
 
     // lambda which returns the current layer
     // (this needs to be reevaluated, since adding a layer can invalidate the reference/pointer due to realloc)
+    // TODO if this needs to be done like that (e.g. for thread safety), why not in the other 10 occurrences of getting the layer in this class??
```

Comment:
remove that comment.
Why its done is written right above it...

Reply:
I mean more the other occurrences in the file. If thread-safety is a concern, I guess the rest of the file(s) have the same problem, but they were accessing the layer directly.

Reply:
ahh.. classical misunderstanding :)

Its not a threading issue, but the fact that down below, e.g. line
https://github.com/OpenMS/OpenMS/pull/5720/files#diff-b7d3caaae7f5a9154973da5f9a1ba05f9f7693d8ee6fd7644fce2880f95c519fR372

will add a new layer, thus potentially invalidating a "static" pointer to the current_layer.

Maybe my comment should be improved...

Reply:
Aaah I even changed it in another function just to be sure 😃 cause I never know where the bug might come from